docs/build/program/apis/sessions.md

```diff
@@ -70,9 +70,11 @@ You can adjust the heartbeat window in the configuration of your machine:
 {{< tabs >}}
 {{% tab name="Builder UI" %}}
 
-On your **Auth/Network** tab, set the **Heartbeat Window**:
+Navigate to your machine's **CONFIGURE** tab in the [Viam app](https://app.viam.com).
+Find the card belonging to the main {{< glossary_tooltip term_id="part" text="part" >}} of your machine at the top of the nested resource tree.
```

Comment:
This might be a little confusing, open to suggestions. It will be at the top of the page in cards and left-hand menu, with the child resources indented off of that